module/VuFind/src/VuFind/Auth/ShibbolethFactory.php

```diff
@@ -60,8 +64,33 @@ public function __invoke(ContainerInterface $container, $requestedName,
         if (!empty($options)) {
             throw new \Exception('Unexpected options sent to factory.');
         }
+        $loader = $this->getConfigurationLoader($container);
+        $request = $container->get('Request');
         return new $requestedName(
-            $container->get(\Laminas\Session\SessionManager::class)
+            $container->get(\Laminas\Session\SessionManager::class),
+            $loader, $request
         );
     }
+
+    /**
+     * Return configuration loader for shibboleth
+     *
+     * @param ContainerInterface $container Service manager
+     *
+     * @return configuration loader
+     */
+    public function getConfigurationLoader(ContainerInterface $container)
+    {
+        $config = $container->get(\VuFind\Config\PluginManager::class)
+            ->get('config');
+        $override = $config->Shibboleth->override;
```

Comment:
Might be good to add a `?? null` on this so notices aren't generated if the config is missing.

Reply:
...but of course if we decide to switch the configuration to a boolean, this will become a non-issue. :-)